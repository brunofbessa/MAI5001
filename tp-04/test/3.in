1 The
1 Book
1 Is
1 On
1 The
1 Table
1 Chair
2 O 
2 Livro
2 Esta
2 Sobre
2 A
2 Cama
2 Mesa
0