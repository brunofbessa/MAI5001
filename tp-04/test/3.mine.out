The O 3 3
Book Livro 2 1
Is Esta 1 2
On Sobre 3 5
The O 3 3
Table Mesa 2 4
<x> 2 5
<x> 5 2
Livro Book 2 1
Esta Is 1 2
Sobre On 4 3
<x> 5 2
<x> 5 2
Mesa Table 3 3
